src/pages/CreateEvent/EventForm.tsx

```diff
@@ -53,7 +53,7 @@ const EventForm: React.FC = () => {
       minParticipants: savedData?.minParticipants || 0,
       maxParticipants: savedData?.maxParticipants || 0,
       judges: savedData?.judges || [{ firstName: "", lastName: "" }],
-      thumbnail: savedData?.thumbnail || null,
+      imageUrl: savedData?.thumbnail || null,
```

Comment:
needs to be changed to imageUrl:savedData?.imageUrl

Reply:
Done